### PR TITLE
Fixes #14

### LINKED ISSUE #14: Issue with ph5tostationxml.py handling empty location codes
@rsdeazevedo @derick-hess 

__Issue description__
After we updated the Seisorz experiment to correctly store empty string location codes, instead of `"--"`, I found a problem with the way the Web Services pass empty location codes to the _ph5tostationxml.py_ API.

The problem was caused because there is not a reliable way, that I know of, to represent empty-string location codes in  a comma separated string. This issue makes it impossible to request the Seisorz experiment that contains only empty location codes, as the PH5toStationXML class expects comma separated strings for location values.

__Fixing the issue required some small changes to ph5tostationxml.py__:
-  I changed the `network_list`, `reportnum_list`, `sta_list`, `array_list`, `location_list`, `channel_list` and `ph5path` parameters in _ph5tostationxml.py_ to expect a Python List object instead of a comma separated string of values.  
- I moved related conversions from comma separated strings to Python List objects to the `__main__` method of _ph5tostationxml.py_, and out of the `PH5toStationXML` class. This way the ability to use the program as a command line application should remain unchanged.

If it is okay with both of you, I am going to submit a pull request with my changes.

Thanks,
Nick

### PR DESCRIPTION
**Pull request by nick-iris**
@derick-hess and @rsdeazevedo please find my changes for issue #14 in this PR and merge if you are okay with the code changes. Thanks.

### CODE REVIEW

**Comment by derick-hess** [comment]
I'll check it out today and merge it.

File: webservices/ph5tostationxml.py [hunk 1]
```diff
@@ -588,9 +571,31 @@ def get_network_date(self):
...
+        args_dict['location_list'] = [x.strip()
+                        for x in args_dict.get('location_list').split(',')]
+    if args_dict.get('channel_list'):
+        args_dict['channel_list'] = [x.strip()
+                        for x in args_dict.get('channel_list').split(',')]
+    if args_dict.get('ph5path'):
+        args_dict['ph5path'] = args_dict.get('ph5path').split(',')
+
+    ph5sxml = PH5toStationXML(args_dict)
+
```
Comment (nick-iris) [inline webservices/ph5tostationxml.py]: Conversions from comma separated strings to Python List objects were moved from spots throughout the PH5toStationXML class into the `__main__` method.

File: webservices/ph5tostationxml.py [hunk 2]
```diff
@@ -154,36 +150,29 @@ def __init__(self, args):
...
         
         nickname = args.get('nickname')
-        if nickname[-3:] == 'ph5':
-
-            PH5FILE = os.path.join(args.get('ph5path'), args.get('nickname'))
-
-        else:
-
-            PH5FILE = os.path.join(args.get('ph5path'), args.get('nickname') + '.ph5')
-
```
Comment (nick-iris) [inline webservices/ph5tostationxml.py]: I removed these lines since `PH5FILE` was not referenced anywhere else in the code.

**Comment by nick-iris** [comment]
Not to be a pain, but could we get this merged today if everything looks alright?

The PH5Beta Web Service is currently broken on the server, and this change to the API along with my other Web Service code changes should fix the problem. I tried to be as minimal  as possible with the code changes.

**Pull request merged**